src/output.rs

```diff
@@ -11,15 +14,38 @@ use crate::{
     line_iterator::LineIterator,
 };
 
+static INIT: Once = Once::new();
```

Comment:
Sorry for all the back and forth: You can move this static into the function body of `configure_pager`, but only the linux version so that the linter is happy on windows too :)

Other than that, lets get this merged

Reply:
Done, nice catch!